tests/playwright/pageobjects/watchlist.page.js

```diff
@@ -0,0 +1,23 @@
+const Page = require( './page' );
+
+class WatchlistPage extends Page {
+	constructor() {
+		super();
+		this.changesList = '.mw-changeslist';
+		this.changeListTitles = '.mw-changeslist-line .mw-title';
+	}
+
+	async open() {
+		await this.openTitle( 'Special:Watchlist' );
+	}
+
+	async getTitles() {
+		await page.waitForSelector( this.changesList );
+		return await ( await page.$( this.changesList ) ).$$eval(
```

Comment:
Why are there double awaits here?

Reply:
try using $eval here

Reply:
@und3fined-v01d, I have split the return code. $eval is works on a single element. Here we are using $$eval() because changesList returns multiple elements and $$eval() works for multiple elements.